app/views/rent_mailer/new_rent_notification.text.erb

```diff
@@ -1,11 +1,7 @@
-Hi, this email has the purpose of notifying you of a new rent.
-It's not necessary to respond this email.
+<%= t('new_rent_notification.header_text') %>
 
-A new rent has been added
-=========================
-User: <%= @rent.user.first_name %> <%= @rent.user.last_name %>
-Rents the book: <%= @rent.book.title %>
-From date: <%= @rent.from %>
-To date: <%= @rent.to %>
-
-Thanks
\ No newline at end of file
+<%= "#{t('new_rent_notification.user_text')}#{@rent.user.first_name} #{@rent.user.last_name}" %>
```

Comment:
I think you didn't apply the changes I requested. The idea is to replace this concatenation here with I18n variables, as shown in this link http://guides.rubyonrails.org/i18n.html#passing-variables-to-translations